crates/fuel-streams/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "fuel-streams"
-authors = { workspace = true }
-keywords = { workspace = true }
-edition = { workspace = true }
-homepage = { workspace = true }
-license = { workspace = true }
-repository = { workspace = true }
-version = { workspace = true }
+authors = ["Fuel Labs <contact@fuel.sh>"]
```

Comment:
Knope has a bug when using `version = { workspace = true }`, so unfortunately we need to repeat basically what we have in the main `Cargo.toml` in the workspace to be able to have it running properly